variants/esp32/sugarcube/platformio.ini

```
@@ -0,0 +1,8 @@
+[env:sugarcube]
+extends = esp32_base
+board = ttgo-lora32-v21
```

```suggestion
board = ttgo-lora32-v21
board_level = extra
```
`board_level` must be set to `extra` :+1: